src/extensions/savetoLaserfiche/GetDocumentDataDialog.tsx

```diff
@@ -311,7 +363,11 @@ function GetDocumentDialogData(props: {
           lfField.isRequired &&
           (!spDocFieldValue || spDocFieldValue.length === 0)
         ) {
-          missingRequiredFields.push(mapping.spField);
+          const currentField: SPProfileConfigurationData | undefined =
+            allSPFieldProperties.find(
+              (prop) => prop.InternalName === mapping.spField.InternalName
```

Comment:
because we need to find the public facing name for that user. This was causing the issue where you saved a field while your SP was in Chinese and then I saw the field in Chinese, even though my SP was in English. I think there's a way to simplify what we save so we are only saving the InternalName/Id but I haven't gotten that far yet.